### ISSUE TITLE
NetworkException silently ignored

### ISSUE THREAD

**Issue by xosofox** [open-telemetry/opentelemetry-php#1112]
Looking at 

https://github.com/opentelemetry-php/sdk/blob/e86a88723207aab73e9ddaea32d2cedddd82a92e/Common/Export/Http/PsrTransport.php#L108

we can see that network errors are silently ignored.

I don't think that is a good practice. E.g. SSL certificate errors on the server side would not be recognized...

Can someone explain what the intention is?
What other options are there?

**Comment by brettmc** [comment]
If there is a network error, the empty catch block causes the code to continue, and retry logic to kick in. Only after the configured number of retries does it bail out with an error (L114 or L123). That error contains a RuntimeException, with the NetworkException stashed in `$previous`, so it would be logged.

That's my understanding from reading the code - have you experienced otherwise?

**Comment by bobstrecansky** [comment]
@xosofox - were you able to solve your problem?  We don't believe this is a bug.

**Comment by xosofox** [comment]
Hi, you are right that "retry logic kicks in" - however in my case, during initial setup with some trial and error -  it was not clear to me that I could identify the issue via the `previous` exception.

Sure, "it works", you end up having an error, but the dev experience for my was not good - I finally succeeded in identifying my error by adding a dump into the empty block, this I started this issue.

Maybe there is a way to highlight the `previous` exception during the retry exception?
